test/functional/change_stream.test.js

```diff
@@ -2772,12 +2783,16 @@ describe('Change Streams', function() {
         });
 
         waitForStarted(changeStream, () => {
-          triggerResumableError(changeStream, () => {
-            events.push('error');
-            coll.insertOne({ x: 2 }, { w: 'majority', j: true }, err => {
-              expect(err).to.not.exist;
-            });
-          });
+          triggerResumableError(
+            changeStream,
+            () => {
+              events.push('error');
+              coll.insertOne({ x: 2 }, { w: 'majority', j: true }, err => {
+                expect(err).to.not.exist;
+              });
+            },
+            250
```

Comment:
Added this small delay because triggering the resumable error _immediately_ after `waitForStarted` causes weird behavior, and isn't particularly realistic.